olm-catalog/serverless-operator/project.yaml

```diff
@@ -46,7 +46,11 @@ dependencies:
   eventing_kafka_broker: knative-v1.7
   # eventing-kafka-broker midstream branch or commit
   eventing_kafka_broker_artifacts_branch: release-v1.7
-  cli: 1.4.1
+  cli: 1.7.1
+  func:
+    util: quay.io/boson/alpine-socat:1.7.4.3-r1-non-root
+    tekton_s2i: registry.redhat.io/ocp-tools-4-tech-preview/source-to-image-rhel8@sha256:98d8cb3a255641ca6a1bce854e5e2460c20de9fb9b28e3cc67eb459f122873dd
+    tekton_buildah: registry.redhat.io/rhel8/buildah@sha256:a1e5cc0fb334e333e5eab69689223e8bd1f0c060810d260603b26cf8c0da2023
```

Comment:
@jrangelramos those are placeholders values for productization tool to replace it. But please let me know if you need different values here.